server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java

```diff
@@ -299,6 +303,60 @@ public void testDeleteOperations() throws Exception {
         }
     }
 
+    public void testUpdateOperations() throws Exception {
+        final String primary = internalCluster().startNode();
+        createIndex(INDEX_NAME);
+        ensureYellow(INDEX_NAME);
```

Comment:
Just for my understanding, why aren't we doing ensureGreen() after starting both primary and replica as we do for rest of tests in this class?

Reply:
Starting with one node ensures that node is the primary. In other tests, any node can hold the primary shard.

Reply:
Makes sense. Thank you!